packages/untp-test-suite/src/core/services/dynamic-loading-schemas/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface IDynamicLoadingSchemaService {
+  (schema: string, version: string): Promise<JSON>;
+}
```

Comment:
The function that uses `IDynamicLoadingSchemaService` doesn't return a JSON type inside the Promise. Instead, it returns an object. I think placing the JSON type here may not be entirely correct.